REPL.cpp

```diff
@@ -99,7 +146,7 @@ bool REPL::ExecuteSwift(std::string line)
     swift::OptionSet<swift::TypeCheckingFlags> type_check_opts;
     swift::performTypeChecking(*src_file, top_level_context, type_check_opts);
     
-    ModifyAST(*src_file);
+    swift::FuncDecl *repl_expr_fn_decl = ModifyAST(*src_file);
```

Comment:
This appears to be unused. Why do you have it and why does ModifyAST now return a `FuncDecl *`?

Reply:
Old code - i'll remove it.